README.md

```diff
@@ -75,77 +75,118 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Fixed issue with editor crashing immediately after displaying zones.
+- Fixed bug when navigating editor options via keyboard where pressing enter on unselected Canvas option launches Grid editor instead.
+- Fixed issue where FancyZones would not restore Console Applications.
+- Fixed Canvas editor and Grid editor window heights. Thanks @niels9001!
+- Fixed crash due to KERNELBASE.dll.
+- Fixed FancyZone icons to be smoother at higher DPI settings. Thanks @niels9001!
+- Fixed crash when changing between zone layouts.
+- Fixed regression where FancyZones does not resize windows on layout change.
+- Adjusted layout settings to reset shortcut key after canceling changes on a particular layout.
+
+### File Explorer add-ons
+
+- Fixed issue where markdown files were still previewed even when "Enable Markdown" was unselected.
+
+### Image Resizer
+
+- Added warning that GIF files with animations may no correctly resize if the encoding used for the files is incompatible.
 
 ### Keyboard Manager
 
-- Fixed screen reader usage bugs to increase intuitiveness.
+- Updated old settings UI style on Keyboard Manager preview in OOBE. Thanks @niels9001!
+- Improved UI for KBM re-mappings list. Thanks @niels9001!
+
+### PowerRename
+
+- Fixed bug where disabling the "Show in default context menu" setting only hides the icon but not the actual option to open PowerRename. Thanks @niels9001!
+- Expanding a plugin option in settings can now be toggled. Thanks @niels9001!
+- Fixed race condition causing PowerRename to crash File Explorer. Thanks @ianjoneill!
 
 ### PowerToys Run
 
-- Fixed crashing bug due to missing image file app.dark.png.
-- Fixed URI plugin bug with handling numeric input. Thanks @davidegiacometti!
-- Improved launch performance of PowerToys run on first call. Thanks @davidegiacometti!
-- Changed URI plugin to launch HTTPS by default instead of HTTP. Thanks @chrisharris333!
-- Added confirmation dialog when system commands are executed from PowerToys Run. Thanks @chrisharris333!
+- Fixed lag caused from PowerToys running in background and invoking Alt-Tab.
+- Resolved file not found exception when loading "System.Windows.Controls.Ribbon".
+- Fixed null reference exception crash.
+- Fixed registry plugin load crash.
+- Fixed unauthorized access exception crash when setting registry keys for the utility.
+- Improved visibility of warnings when plugin properties are collapsed. Thanks @niels9001!
+- Added search for Plugin Manager. Thanks @davidegiacometti!
+- Fixed VSCode workspace plugin not working. Thanks @BenConstable9!
 
 ### Video Conference Mute
 
 - Fixed toolbar top right vertical offset to allow users to close other app windows.
 - Fixed compatibility issues for certain systems when compiling from source.
+- Fixed toolbox from persisting on screen.
+- Fixed microphone un-muting when changing Video Conference Mute toolbar position.
+- Added Video Conference Mute to OOBE.
```

Comment:
It is not visible on this version, so this is not correct.

Reply:
These changes will go live next week with v0.46 as mentioned earlier. We can take this information out of the release notes, but here in the README should be fine.

Reply:
At this point these are changes explicitly for 0.45, not for 0.46...